pom.xml

```diff
@@ -36,6 +44,51 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>1.8</version>
+                <executions>
+                    <execution>
+                        <phase>generate-resources</phase>
+                        <configuration>
+                            <tasks>
+                                <echo message="unzipping tools.jar" />
+                                <unzip src="${java.home}/../lib/tools.jar" dest="${project.build.directory}/classes" />
+                            </tasks>
+                        </configuration>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <archive>
+                                <manifest>
+                                    <mainClass>
+                                        com.codice.acdebugger.Main
+                                    </mainClass>
+                                </manifest>
+                                <manifestEntries>
+                                    <Class-Path>tools.jar</Class-Path>
```

Comment:
❓ Do you need this reference since you're unzipping the tools jar in the `generate-resources` phase?